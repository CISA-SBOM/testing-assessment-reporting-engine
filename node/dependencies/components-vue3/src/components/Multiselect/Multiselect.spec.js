import { describe, expect, it } from 'vitest'
import { shallowMount } from '@vue/test-utils'
import Component from "./Multiselect.vue";

describe("Multiselect.vue", () => {
  it("is a Vue instance", () => {
    const wrapper = shallowMount(Component);
    expect(wrapper.vm).toBeTruthy();
  });

  it("matches snapshot with no props assigned", () => {
    const props = {};
    const wrapper = shallowMount(Component, { props });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
