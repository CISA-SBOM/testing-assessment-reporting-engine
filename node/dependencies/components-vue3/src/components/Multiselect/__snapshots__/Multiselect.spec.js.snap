// Vitest Snapshot v1

exports[`Multiselect.vue > matches snapshot with no props assigned 1`] = `
"<div data-id=\\"sds-multiselect\\" class=\\"sds-multiselect canSearch\\" data-v-00d7dde3=\\"\\">
  <ul class=\\"tag-list single\\" data-v-00d7dde3=\\"\\">
    <li style=\\"width: 0px;\\" class=\\"tag-list-item input\\" data-v-00d7dde3=\\"\\"><span class=\\"faux-input\\" aria-hidden=\\"true\\" data-v-00d7dde3=\\"\\"></span>
      <!----><input placeholder=\\"\\" style=\\"width: 0px;\\" autocapitalize=\\"off\\" autocomplete=\\"off\\" spellcheck=\\"false\\" autocorrect=\\"off\\" type=\\"text\\" class=\\"p-0 m-0 border-0 focus:shadow-none focus:ring-0\\" data-v-00d7dde3=\\"\\">
    </li>
  </ul>
  <!---->
  <div aria-hidden=\\"true\\" class=\\"multiselect-caret\\" data-v-00d7dde3=\\"\\"></div>
  <!---->
</div>"
`;
